### ISSUE TITLE
NEM: CLIP Prompt Ensembling for Surveillance Context

### ISSUE THREAD

**Issue by mikesvoboda** [mikesvoboda/nemotron-v3-home-security-intelligence#3495]
### Objective

Implement prompt ensembling for CLIP to improve classification accuracy by 5-10%.

### Implementation

**File:** `ai/clip/model.py`

1. Create surveillance-specific prompt templates:

   ```python
   SURVEILLANCE_TEMPLATES = [
       "a surveillance camera photo of {}",
       "a security camera image showing {}",
       "a CCTV footage frame of {}",
       "a home security camera view of {}",
       "a night vision camera image of {}",
   ]
   ```
2. Update classification method to use ensembling:

   ```python
   def classify_with_ensemble(self, image, labels):
       all_embeddings = []
       for template in SURVEILLANCE_TEMPLATES:
           prompts = [template.format(label) for label in labels]
           embeddings = self.encode_text(prompts)
           all_embeddings.append(embeddings)
       # Average across templates
       ensemble_embeddings = torch.mean(torch.stack(all_embeddings), dim=0)
       return self.compute_similarity(image, ensemble_embeddings)
   ```
3. Add configurable template selection based on camera type

### Acceptance Criteria

- [ ] New `classify_with_ensemble` method implemented
- [ ] Configurable template selection
- [ ] Unit tests comparing ensemble vs single-prompt accuracy
- [ ] Benchmark showing accuracy improvement
- [ ] API endpoint updated to use ensembling by default

### Design Reference

See `docs/plans/2026-01-19-model-zoo-prompt-improvements-design.md` Section 3.2
